new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/KubernetesExecFactory.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions.impl.primitive;
+
+import io.kubernetes.client.openapi.models.V1Pod;
```

Comment:
way to much OOP going on here -- use the simple functional pattern with static methods and fluent api instead of all this

Reply:
Simplified the implementation to be more functional.